addons/building_markers/functions/fnc_set.sqf

```diff
@@ -61,6 +61,8 @@ if (_set) then {
     _marker setMarkerDir getDir _object;
     _object setVariable [QGVAR(marker), _marker, true];
 
+    [QEGVAR(common,setMarkerDrawPriority), [_marker, -1], _marker] call CBA_fnc_globalEventJIP;
```

Comment:
Need to handle removing the JIP event if the marker is deleted.

Reply:
Should building markers be removable for other reasons than building deletion?
Do building markers remain global or should local changes be permitted?

If not, I imagine it's possible to replace the removed marker by detecting when a marker has been deleted with [this EH](https://community.bistudio.com/wiki/Arma_3:_Mission_Event_Handlers#MarkerDeleted):
- You could add this EH on all clients and they would raise an event on the server which would handle the deletion or replacement of the marker, depending if the marker was deleted because the building was deleted or for another reason.
- You could add this EH on the server only and have it deal with the marker deletion or replacement. This assumes that all building marker deletions would be global though.

If building markers should be removable, I guess the EH linked above still can be used, but it would be a bit more tricky: What if `deleteMarkerLocal` is used on building markers? Should the marker be replaced, so that you force people to use `deleteMarker` instead, or should the marker be editable on each client individually, as in permit different marker settings (existence, visibility, size, direction, etc)?

Reply:
> nice one [3ea5c51](https://github.com/zen-mod/ZEN/pull/733/commits/3ea5c510db6156c575a069fb72dd9f61dd2b24c7) I don't think any mod can be so protective of its stuff. What use case is there for building markers showing differently between clients?

I agree that a mod shouldn't be that overprotective. As for the use case of building markers showing differently on clients, I don't really know, I'm not a mission maker. I was mostly thinking of how badly the marker system could/would break if you started messing with markers on a per client basis.